webrecorder/browsertrix
# Sort Collections by size

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/1014



### Changes

- Adds "Size" column to Collections list
- Adds "Size" option to sort dropdown

### Manual testing

1. Log in
2. Click "Collections". Verify size for each Collection is shown in list
3. Click "Sort by" filter and select "Size". Verify list updates to be sorted by size

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Collections list | <img width="1125" alt="Screenshot 2023-07-31 at 4 27 30 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/1f761ce1-c82f-4e2c-9c70-37a42af77e47"> |


### Follow-ups

Should we display Collection info like size and page count in the Collection detail view?
